peer/network.go

```diff
@@ -543,8 +544,12 @@ func (n *network) TrackBandwidth(nodeID ids.NodeID, bandwidth float64) {
 	n.peers.TrackBandwidth(nodeID, bandwidth)
 }
 
-func (n *network) NewAppProtocol(protocol uint64, handler p2p.Handler, options ...p2p.ClientOption) (*p2p.Client, error) {
-	return n.network.NewAppProtocol(protocol, handler, options...)
+func (n *network) NewClient(protocol uint64, options ...p2p.ClientOption) (*p2p.Client, error) {
+	return n.network.NewClient(protocol, options...)
+}
+
+func (n *network) AddHandler(protocol uint64, handler p2p.Handler) error {
+	return n.network.AddHandler(protocol, handler)
```

Comment:
can we rename the inner `network `struct (p2p)?

Reply:
Or alternatively maybe we can directly embed `*p2p.Network` in to the struct.

Reply:
Embedding makes this look weird still sine the name is then `n.Network`... i'll just rename this to something else